src/GitHub.VisualStudio.UI/UI/Controls/InfoPanel.xaml.cs

```diff
@@ -80,7 +80,9 @@ public InfoPanel()
         {
             InitializeComponent();
 
-            this.DataContext = this;
+            DataContext = this;
```

Comment:
Also removed the `this` keyword here, since we don't need it here (also VS suggested we do it).